app/views/users/sessions/email_not_found.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Your school has not registered</h2>
+    <h1 class="govuk-heading-l">Your school has not registered</h1>
```

Comment:
Just curious why you made this change? Is h2 more accessible?

Reply:
It's changing from a h2 to an h1, because the page didn't have an h1 and every page should have one